pkg/systemstatsmonitor/cpu_collector.go

```diff
@@ -50,27 +48,17 @@ type cpuCollector struct {
 	mSystemProcsRunning    *metrics.Int64Metric
 	mSystemProcsBlocked    *metrics.Int64Metric
 	mSystemInterruptsTotal *metrics.Int64Metric
+	mSystemCPUStat         *metrics.Float64Metric // per-cpu time from /proc/stats
 
 	config *ssmtypes.CPUStatsConfig
 
 	lastUsageTime map[string]float64
 }
 
 func NewCPUCollectorOrDie(cpuConfig *ssmtypes.CPUStatsConfig) *cpuCollector {
-	cc := cpuCollector{tags: map[string]string{}, config: cpuConfig}
```

Comment:
Why removing the kernel and os labels? Can you explain it in the PR description?

Reply:
Ah I meant to but forgot amongst oncall chaos. :) Will add now